### PR TITLE
[1.4 - DO NOT MERGE] Add nproc limits content

### LINKED ISSUE #358: Setting nproc limit for Diego cell
https://trello.com/c/XvU6E5Vg/955-allow-bumping-up-the-nproc-limit-for-vcap-user

Marking as blocked until the card itself is unblocked.

### PR DESCRIPTION
**Pull request by btat**
Resolves #358

### CODE REVIEW

File: xml/cap_admin_nproc_limits.xml [hunk 1]
```diff
@@ -0,0 +1,129 @@
...
+   <dm:bugtracker></dm:bugtracker>
+   <dm:translation>yes</dm:translation>
+  </dm:docmanager>
+ </info>
+
+ <para>
+  In &productname; there are parameters,
+  <command>kube.limits.nproc.soft</command> and
+  <command>kube.limits.nproc.hard</command>, to configure a soft nproc limit
+  and a hard nproc limit for <literal>scf</literal> pods. By default, these
```
Comment (cjschroder) [inline xml/cap_admin_nproc_limits.xml]: Maybe add a brief definition of nproc

File: xml/cap_admin_nproc_limits.xml [hunk 2]
```diff
@@ -0,0 +1,129 @@
...
+[
+  <!ENTITY % entities SYSTEM "entity-decl.ent">
+    %entities;
+]>
+<chapter version="5.0" xml:id="cha.cap.manage-nproc-limits"
+  xmlns="http://docbook.org/ns/docbook"
+  xmlns:xi="http://www.w3.org/2001/XInclude"
+  xmlns:xlink="http://www.w3.org/1999/xlink">
+ <info>
+  <title>Managing nproc limits of pods</title>
```
[the remark this answers is not in the document]
Comment (btat) [reply xml/cap_admin_nproc_limits.xml]: Updated in https://github.com/SUSE/doc-cap/pull/414/commits/0e6e35aeffbbf7c98e87eec1312c3ffc5822f6bd

File: xml/cap_admin_nproc_limits.xml [hunk 3]
```diff
@@ -0,0 +1,129 @@
...
+
+<screen>kube:
+  limits:
+    nproc:
+      hard: ""
+      soft: ""
+</screen>
+
+ <para>
+   If <literal>uaa</literal> is deployed, pass your <literal>uaa</literal>
```
[the remark this answers is not in the document]
Comment (btat) [reply xml/cap_admin_nproc_limits.xml]: Updated in https://github.com/SUSE/doc-cap/pull/414/commits/0e6e35aeffbbf7c98e87eec1312c3ffc5822f6bd

File: xml/cap_admin_nproc_limits.xml [hunk 4]
```diff
@@ -0,0 +1,129 @@
...
+
+<screen>&prompt.user;watch -c 'kubectl get pods --namespace scf'</screen>
+
+ <para>
+  When all pods are in a ready state, do the following to confirm the limits
+  have been configured correctly:
+ </para>
+
+ <para>
+  Open a shell into any container. The below opens a shell to the default
```
[the remark this answers is not in the document]
Comment (btat) [reply xml/cap_admin_nproc_limits.xml]: Updated in https://github.com/SUSE/doc-cap/pull/414/commits/0e6e35aeffbbf7c98e87eec1312c3ffc5822f6bd

File: xml/cap_admin_nproc_limits.xml (hunk 1 above)
Comment (btat) [reply xml/cap_admin_nproc_limits.xml]: Updated in https://github.com/SUSE/doc-cap/pull/414/commits/0e6e35aeffbbf7c98e87eec1312c3ffc5822f6bd

File: xml/cap_admin_nproc_limits.xml [hunk 5]
```diff
@@ -0,0 +1,209 @@
...
+<screen>&prompt.user;ulimit -u
+
+&prompt.user;cat /etc/security/limits.conf | grep nproc
+</screen>
+    </step>
+   </procedure>
+  </sect2>
+ 
+  <sect2 xml:id="sec.cap.existing-deployment-apply-nproc">
+   <title>Existng Deployments</title>
```
Comment (cjschroder) [inline xml/cap_admin_nproc_limits.xml]: Existing

File: xml/cap_admin_nproc_limits.xml [hunk 6]
```diff
@@ -0,0 +1,209 @@
...
+ </info>
+
+ <para>
+  Nproc is the maximum number of processes allowed per user. In the case of
+  <literal>scf</literal>, the nproc value applies to the
+  <literal>vcap</literal> user and has a default limit of 4096. In
+  <literal>scf</literal>, there are parameters,
+  <command>kube.limits.nproc.soft</command> and
+  <command>kube.limits.nproc.hard</command>, to configure a soft nproc limit
+  and a hard nproc limit for processes spawn by the <literal>vcap</literal>
```
Comment (cjschroder) [inline xml/cap_admin_nproc_limits.xml]: spawned

File: xml/cap_admin_nproc_limits.xml (hunk 5 above)
Comment (btat) [reply xml/cap_admin_nproc_limits.xml]: Thanks, fixed

File: xml/cap_admin_nproc_limits.xml (hunk 6 above)
Comment (btat) [reply xml/cap_admin_nproc_limits.xml]: Thanks, fixed

**Comment by cjschroder** [comment]
:+1:  Luvverly!

**Pull request merged**